tswow-core/Public/global.d.ts

```diff
@@ -177,6 +177,17 @@ declare const enum DeathStatus {
     DEAD = 2
 }
 
+declare const enum LineOfSightChecks {
+    LINEOFSIGHT_CHECK_VMAP      = 0x1, // check static floor layout data
+    LINEOFSIGHT_CHECK_GOBJECT   = 0x2, // check dynamic game object data
+    LINEOFSIGHT_ALL_CHECKS      = (LINEOFSIGHT_CHECK_VMAP | LINEOFSIGHT_CHECK_GOBJECT)
+}
+
+declare const enum VMapModelIgnoreFlags {
+    Nothing = 0x00,
+    M2      = 0x01
+}
```

Comment:
check if you can read these automatically from trinitys source code, you can look at the other enum declarations at the top of the file for reference. essentially you use syntax like this:

```ts
declare const enum LineOfSightChecks {} /** File.h:TrinityEnumName */
```